Sources/Controllers/2.0/Settings/ImportExport/OABaseSettingsListViewController.mm

```diff
@@ -362,10 +365,12 @@ - (UITableViewCell *) tableView:(nonnull UITableView *)tableView cellForRowAtInd
                 }
                 cell.openCloseGroupButton.tag = indexPath.section << 10 | indexPath.row;
                 [cell.openCloseGroupButton addTarget:self action:@selector(openCloseGroupButtonAction:) forControlEvents:UIControlEventTouchUpInside];
-                
+
                 cell.selectionButton.tag = indexPath.section << 10 | indexPath.row;
                 [cell.selectionButton addTarget:self action:@selector(onGroupCheckmarkPressed:) forControlEvents:UIControlEventTouchUpInside];
-                
+                cell.selectionGroupButton.tag = indexPath.section << 10 | indexPath.row;
+                [cell.selectionGroupButton addTarget:self action:@selector(onGroupCheckmarkPressed:) forControlEvents:UIControlEventTouchUpInside];
```

Comment:
removeTarget must be called before addTarget at cellForRowAtIndexPath.
Needs to be fixed in all other places! @PaulStets